app/src/main/java/com/github/se/gomeet/ui/mainscreens/create/CreateEvent.kt

```diff
@@ -26,33 +27,36 @@ import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.text.TextStyle
 import androidx.compose.ui.text.font.Font
 import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontStyle
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.text.style.TextAlign
-import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
-import androidx.navigation.compose.rememberNavController
 import com.github.se.gomeet.R
 import com.github.se.gomeet.ui.navigation.BottomNavigationMenu
 import com.github.se.gomeet.ui.navigation.NavigationActions
 import com.github.se.gomeet.ui.navigation.Route
 import com.github.se.gomeet.ui.navigation.TOP_LEVEL_DESTINATIONS
+import com.github.se.gomeet.ui.theme.DarkerCyan
+import com.github.se.gomeet.ui.theme.Grey
 import com.github.se.gomeet.viewmodel.EventViewModel
 import java.time.LocalDate
 import java.time.format.DateTimeFormatter
 import java.time.format.DateTimeParseException
 
 @Composable
-fun PublicCreate(nav: NavigationActions, eventViewModel: EventViewModel) {
+fun CreateEvent(nav: NavigationActions, eventViewModel: EventViewModel, isPrivate: Boolean) {
 
   val titleState = remember { mutableStateOf("") }
   val descriptionState = remember { mutableStateOf("") }
   val locationState = remember { mutableStateOf("") }
   val textDate = remember { mutableStateOf("") }
   var dateState by remember { mutableStateOf<LocalDate?>(null) }
   var dateFormatError by remember { mutableStateOf(false) }
-  val price by remember { mutableDoubleStateOf(0.0) }
+  var price by remember { mutableDoubleStateOf(0.0) }
+  var priceText by remember { mutableStateOf("") }
   val url = remember { mutableStateOf("") }
+  val isPrivateEvent = remember { mutableStateOf(false) }
 
```

Comment:
we should not forget that users should be able to upload images and add tags to their events